pages/admin/permit-holders.tsx

```diff
@@ -1,16 +1,344 @@
+  const expired = DateUtils.isPastDay(value.expiryDate);
+  const expiresSoon = DateUtils.isDayInRange(today, {
+    from: today,
+    to: DateUtils.addMonths(today, 1),
+  });
+  return (
+    <Flex>
+      <Text as="span" mr="9px">
+        {value.appNumber}
+      </Text>
+      {expired && <WarningTwoIcon color="secondary.critical" />}
+      {expiresSoon && <WarningIcon color="secondary.caution" />}
+    </Flex>
+  );
+}
+
+type DropDownItemsProps = {
+  readonly items: Array<string>;
+  readonly setStateCallback: Dispatch<SetStateAction<string>>;
+};
+
+function DropDownItems({ items, setStateCallback }: DropDownItemsProps) {
+  return (
+    <>
+      {items.map((value, i) => (
+        <MenuItem
+          key={`dropDownItem-${i}`}
+          onClick={() => {
+            setStateCallback(value);
+          }}
+        >
+          {value}
+        </MenuItem>
+      ))}
+    </>
+  );
+}
+
+type MenuTextProps = {
+  readonly name: string;
+  readonly value?: string;
+};
+
+function MenuText({ name, value }: MenuTextProps) {
+  return (
+    <>
+      <Text as="span" textStyle="button-semibold">
+        {`${name}: `}
+      </Text>
+      <Text as="span" textStyle="button-regular">
+        {value}
+      </Text>
+    </>
+  );
+}
+
+function DayPickerStyling() {
+  return (
+    <Helmet>
+      <style>{`
+        .Selectable .DayPicker-Day--selected:not(.DayPicker-Day--start):not(.DayPicker-Day--end):not(.DayPicker-Day--outside) {
+          background-color: #f0f8ff !important;
+          color: #4a90e2;
+        }
+        .Selectable .DayPicker-Day {
+          border-radius: 0 !important;
+        }
+        .Selectable .DayPicker-Day--start {
+          border-top-left-radius: 50% !important;
+          border-bottom-left-radius: 50% !important;
+        }
+        .Selectable .DayPicker-Day--end {
+          border-top-right-radius: 50% !important;
+          border-bottom-right-radius: 50% !important;
+        }
+      `}</style>
+    </Helmet>
+  );
+}
 
 // Internal home page
 export default function PermitHolders() {
+  const [permitStatusFilter, setPermitStatusFilter] = useState('All');
+  const [userStatusFilter, setUserStatusFilter] = useState('All');
+  const [range, setRange] = useState<RangeModifier>({ from: undefined, to: undefined });
+
+  const permitStatusOptions = [
+    'All',
+    'In Progress',
+    'Completed',
+    'Pending',
+    'Expiring Soon',
+    'Expired',
+    'Rejected',
+  ];
+  const userStatusOptions = ['All', 'Active', 'Inactive'];
+
+  const handleDayClick: DayPickerProps['onDayClick'] = day => {
+    setRange(DateUtils.addDayToRange(day, range));
+  };
+
+  const modifier = { start: range.from || undefined, end: range.to || undefined };
+
+  const dateRangeString = () => {
+    if (!range.to) {
+      return 'YYYY-MM-DD - YYYY-MM-DD';
+    }
+    if (range.from && range.to) {
+      return `${range.from.toLocaleDateString('en-CA')} - ${range.to.toLocaleDateString('en-CA')}`;
+    }
+  };
+
```

Comment:
Not sure if we could do this with react date picker, but when I was playing around with the date picker, I found it a bit unintuitive that I couldn't easily change the first date picked. For example, if a user selected this at first:
![image](https://user-images.githubusercontent.com/60051956/127359242-113730b1-1beb-4cbe-94d0-7dd3d89f4eaf.png)

But then realized they made a mistake and meant to select July 19 instead of July 17 (first date in the range), the date picker I think defaults to modifying the second date when I click on July 19:
![image](https://user-images.githubusercontent.com/60051956/127359544-b84d28c0-63b5-40d2-a8fb-fa4445799d05.png)

Is there a way to specify which date to change?

Reply:
Also, we could maybe consider adding some logic that won't allow the user to select a date prior to present day. Don't think that's super important though

Reply:
Both great points, to keep this PR short i think it's best to make another ticket for that and it can be addressed after the api hookup or post-mvp. For posterity here is a good example on the react-day-picker docs that we can follow to do your first suggestion http://react-day-picker.js.org/examples/selected-range-enter

Reply:
cc @OustanDing just so you're in the loop